### ISSUE TITLE
Support config_path for cron expressions

### ISSUE THREAD

**Issue by peterjaap** [Ethan3600/EthanYehuda_CronjobManager#201]
While working on #97, I discovered that this extension does not show the frequency for cron jobs that use a `config_path` field instead of a `schedule` field.

For example the `currency_rates_update` job;

![image](https://github.com/Ethan3600/magento2-CronjobManager/assets/431360/f98a3e12-6fc5-4389-92dd-13ac51629b0b)

It's crontab.xml looks like this;

```
<config xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:module:Magento_Cron:etc/crontab.xsd">
    <group id="default">
        <job name="currency_rates_update" instance="Magento\Directory\Model\Observer" method="scheduledUpdateCurrencyRates">
            <config_path>crontab/default/jobs/currency_rates_update/schedule/cron_expr</config_path>
        </job>
    </group>
</config>
```

**Comment by peterjaap** [comment]
Added here; https://github.com/Ethan3600/magento2-CronjobManager/pull/203

You can test this by setting the currency_rates_update cron expression through magerun2;

```
magerun2 config:store:set crontab/default/jobs/currency_rates_update/schedule/cron_expr "0 * * * *"
magerun2 cache:clear config
```

**Comment by fredden** [comment]
This has been completed in https://github.com/Ethan3600/magento2-CronjobManager/pull/203, which landed in [v2.1.6](https://github.com/Ethan3600/magento2-CronjobManager/releases/tag/v2.1.6).
